docs/Demo-App.md

```diff
@@ -60,6 +60,7 @@ If you want to configure the application for online service, then follow the nex
    yarn android
    yarn pods
    yarn ios
+   yarn ios # repeat ios, to detect the changes properly
```

Comment:
Hmm 🤔

Reply:
Yes, one build is not enough 😀 It sounds like some cocoapods magic...